src/style.css

```diff
@@ -1 +1,69 @@
 /* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  margin: 0;
+}
```

Comment:
reset tag which u need like you did on the lesson with Ihor
move fonts to html selector